docs/docs/04-multiple-components.ru-RU.md

```diff
@@ -0,0 +1,182 @@
+---
```

Comment:
We’d also need to change `03-interactivity-and-dynamic-uis.ru-RU.md` to point to this page as its `next`.